KomaMRIIO/src/KomaMRIIO.jl

```diff
@@ -0,0 +1,29 @@
+module KomaMRIIO
+
+using KomaMRICore
+
+# General
+using Reexport, Scanf, Pkg
+
+# IO
+using FileIO, HDF5, MAT, JLD2
+
+using MRIBase, MRIFiles
+@reexport using MRIBase: Profile, RawAcquisitionData, AcquisitionData, AcquisitionHeader
+@reexport using MRIFiles: ISMRMRDFile
+
+include("io/Pulseq.jl")
+include("io/JEMRIS.jl")
+include("io/MRiLab.jl")
+include("io/ISMRMRD.jl")
+
+# Pulseq
+export read_seq
+
+# ISMRMRD
+export signal_to_raw_data
+
+# Phantom
+export read_phantom_jemris, read_phantom_MRiLab
+
```

Comment:
This is missing a `KomaMRIIO.__VERSION__` variable (This is why `Pkg` is being imported).

Reply:
You are right. It is going to be added.